sphinx_doc/howtos/tutorials/docs/navigation2_on_real_turtlebot3.rst

```diff
@@ -0,0 +1,161 @@
+
+1- Launch Turtlebot 3 Robot State Publisher
+-------------------------------------------
+
+Launch Turtlebot 3 robot state publisher,
+
+  ``ros2 launch turtlebot3_bringup turtlebot3_state_publisher.launch.py use_sim_time:=False``
+
+Make sure use_sim_time is set to **False**.
+
+2- Launch Navigation2
+---------------------
+
+You need to have a map of the environment where you want to Navigate Turtlebot 3. Required files:
+
+   - ``your-map.map``
+   - ``your-map.yaml``
+
+``<your_map>.yaml`` is the configuration file for the map we want to provide Navigation2.
+In this case, it has the map resolution value, threshold values for obstacles and free spaces, and a map file location.
+You need to make sure these values are correct. 
+More information about the map.yaml can be found `here <http://wiki.ros.org/map_server>`_.
+
+Launch Navigation 2. If you set autostart:=False, you need to click on the start button in RViz to initialize the nodes. 
+Make sure `use_sim time` is set to **False**, because we want to use the system time instead of the time simulation time from Gazebo.
+
+``ros2 launch nav2_bringup nav2_bringup_launch.py use_sim_time:=False autostart:=False map:=/path/to/your-map.yaml``
+
+Don't forget to change **/path/to/your-map.yaml** to the actual path to the your-map.yaml file.
+
+3-  Launch RVIZ
+---------------
+
+Launch RVIZ with a pre-defined configuration file.
+
+  ``ros2 run rviz2 rviz2 -d $(ros2 pkg prefix nav2_bringup)/share/nav2_bringup/launch/nav2_default_view.rviz``
+
+Now, you should see a shadow of Turtlebot 3 robot model in the center of the plot in Rviz.
+Click on the Start button (Bottom Left) if you set the auto_start parameter to false.
+
+.. image:: images/Navigation2_on_real_Turtlebot3/rviz_after_launch_view.png
+    :height: 720px
+    :width: 1024px
+    :alt: RViz after launch, auto_start = False
+
+Now, the map should appear in RViz.
+
+.. image:: images/Navigation2_on_real_Turtlebot3/rviz_slam_map_view.png
+    :height: 720px
+    :width: 1024px
+    :alt: A map generated by using SLAM in RViz
+
+4- Initialize the Location of Turtlebot 3
+-----------------------------------------
+
+First, find where the robot is on the map. Check where your robot is in the room.
+
+Set the pose of the robot in RViz. 
+Click on the 2D Pose Estimate button and point the location of the robot on the map. 
+The direction of the green arrow is the orientation of Turtlebot.
+
+.. image:: images/Navigation2_on_real_Turtlebot3/rviz_set_initial_pose.png
+    :height: 720px
+    :width: 1024px
+    :alt: Set initial pose in RViz
+
+Now, the 3D model of Turtlebot should move to that location. 
+A small error in the estimated location is tolerable.
+
+5-  Send a Goal Pose
+--------------------
+
+Pick a target location for Turtlebot on the map. 
+You can send Turtlebot 3 a goal position and a goal orientation by using the **Navigation2 Goal** and the **GoalTool** buttons.
+
+*Note*: Difference between buttons. 
```

Comment:
`Note: Navigation2 Goal button uses a ROS2 Action to send the goal and GoalTool publishes the goal to a topic.`

I dont think the prefix of `Difference between buttons` is required.